.github/workflows/check-self.yml

```diff
@@ -31,8 +31,8 @@ jobs:
         key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}
     - name: Check Self
       run : |
-        make docker-check-self
+        PYTEST_OPTS='-n 8' make docker-check-self
```

Comment:
I'm surprised that the GHA runners support this many tasks in parallel, good find :+1: